config/sample_manifest_excel/ranges.yml

```diff
@@ -85,3 +85,8 @@ collected_by_for_cardinal:
 collected_by_for_controls:
   options:
     - Sanger Institute
+retention_instruction_options:
+  options:
+    - Destroy after 2 years
+    - Return to customer after 2 years
+    - Long term storage
```

Comment:
Doesn't really matter, but it looks like all the other fields (apart from Harry's recent one, because that's got 2 ranges for 1 field), follow a naming convention where the yml key is just the field name? i.e. `retention_instruction:`